bot/cmd_trade.go

```diff
@@ -0,0 +1,35 @@
+// added by Lewymd, if it breaks it's his fault
+
+package bot
+
+import (
+	"fmt"
+
+	"github.com/bwmarrin/discordgo"
+)
+
+// handleHelpCmd handles the "trade" command
```

Comment:
should probably update the function name here as well, and remove the leading space :)